cmd/start.go

```diff
@@ -84,3 +92,27 @@ func startHandler(cmd *cobra.Command, _ []string) {
 
 	logs.Info().Msg("shutting down timelock-worker")
 }
+
+func healthHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
The health itself doesn't check any condition in timelock that requires it to be alive, the subscription could be dead and the web server still reporting ok.